addons/project/views/project_portal_templates.xml

```diff
@@ -146,7 +146,11 @@
                             <th t-if="groupby == 'status'" t-attf-colspan="{{grouped_tasks_colspan}}">
                                 <span class="text-truncate" t-field="tasks[0].sudo().kanban_state"/></th>
                             <th t-if="groupby == 'customer'" t-attf-colspan="{{grouped_tasks_colspan}}">
-                                <span class="text-truncate" t-field="tasks[0].sudo().partner_id.name"/></th>
+                                <span t-if="tasks[0].sudo().partner_id.name"
```

Comment:
```suggestion
                                <span t-if="tasks[0].sudo().partner_id"
```
No need to check the name here

Reply:
> No need to check the name here

@mash-odoo on holiday so i done the changes.thanks